cluster/charts/crossplane/README.md.gotmpl

```diff
@@ -93,7 +93,7 @@ replicas: 1
 deploymentStrategy: RollingUpdate
 
 image:
-  repository: crossplane/crossplane
+  repository: xpkg.upbound.io/crossplane/crossplane
```

Comment:
i'm not actually sure what this `README.md.gotmpl` file is used for or if there is any automation around it, any ideas?

Reply:
Yes, it's automatically rendered to the actual README of the chart with all the values updated, this part feels a bit superfluous though, we could remove it